corona-warn-app/cwa-app-android
# PDF export for Certificates (EXPOSUREAPP-8900)

## Testing
1. generate and add some tc, vc, rc
2. open certificate detail and click "Druckversion anzeigen" from top app bar menu click "Weiter"
3. check the PDF preview, try to share/print it and check formatting and data in the file
